src/containers/Review/ReviewPage.tsx

```diff
@@ -48,9 +48,11 @@ const ReviewPage: React.FC<{
   fullApplicationStructure: FullStructure
 }> = ({ reviewAssignment, previousAssignment, fullApplicationStructure }) => {
   const { strings } = useLanguageProvider()
+  const { serial } = fullApplicationStructure.info
   const {
     userState: { currentUser },
   } = useUserState()
+  const [reviewStatus, setReviewStatus] = useState(ReviewStatus.Submitted)
```

Comment:
Not entirely sure why this needs to be a state variable. Can't it just be derived from `reviewStructure` at any time?

I think it works to just declare it after the `useGetReviewStructureForSections` hook:

```ts
const { reviewStructure, error } = useGetReviewStructureForSections({
    reviewAssignment,
    reviewStructure: fullApplicationStructure,
  })

const reviewStatus = reviewStructure?.thisReview?.current.reviewStatus ?? ReviewStatus.Submitted
```

It'll still get updated whenever the value in `reviewStatus` changes (as reviewStatus will be updated from the hook).

Reply:
Yes, good call. I made this change earlier when I didn't realise there was a problem about the changeStatus action not running correctly. So I haven't change this after to check that. I'll change this back as you suggested. 👍